Source/dump.f90

```diff
@@ -3030,7 +3030,7 @@ SUBROUTINE INITIALIZE_DIAGNOSTIC_FILE(DT)
       IF (SF%VEG_LSET_IGNITE_T<1.E6_EB) &
       WRITE(LU_OUTPUT,'(A,ES9.2)')  '     Ignition Time (s)           ', SF%VEG_LSET_IGNITE_T
       WRITE(LU_OUTPUT,'(A,ES10.3)') '     Burn Duration (s)           ', SF%BURN_DURATION
-      WRITE(LU_OUTPUT,'(A,ES10.3)') '     Rate of Spread (m/s)        ', SF%VEG_LSET_ROS
+      WRITE(LU_OUTPUT,'(A,ES10.3)') '     Rate of Spread,R00 (m/s)    ', SF%VEG_LSET_ROS00
```

Comment:
Can this be VEG_LSET_ROS_0?

Rate of Spread, R_0 (m/s)

Reply:
The double zero, _00, denotes zero wind, zero slope. The zero wind, zero slope, rate of spread shows up in a number of models, the Rothermel model being one. Papers have the _00 notation. For this reason, I'd like to keep the _00 notation.

Reply:
OK, thanks.  Can we change to ROS_00 (underscore), add code comments, and add space to output `Rate of Spread, ROS_00`?